doc/SCALING.md

```diff
@@ -89,11 +66,33 @@ resources:
 
 Initially there is only a single `stunner` instance in the cluster. As new calls start to come in the amount of utilized CPU is increasing.  
 
-Scaling out is triggered when CPU usage of the `stunner` pod reaches 1500m (three times the requested CPU) core CPU.
+Scale out is triggered when CPU usage of the `stunner` pod reaches 1500m (three times the requested CPU) core CPU.
 
 More calls come in and the summarized CPU usage by `stunner` pods reach 3000m core CPU, this is 1500m core on average. Scale out happens again. 
 
-As the the meeting is ended and participants leave the room the load will drop and the average CPU will fall under 3000m. Scaling in happens as the `HPA` removes one of the `stunner` instances. And so on...
+As participants leave the room, the load will drop and the average CPU will fall under 3000m. Scaling in happens as the `HPA` removes one of the `stunner` instances. And so on...
+
+###  Scaling-up (out)
+
+When adding new instances to the existing `stunner` replica set we don't have much to worry about. The infrastructure for existing calls won't get interrupted, it stays the same as before the upscale event. Only when new user calls come in the cloud loadbalancer have an additional endpoint to choose from. This way we can achieve that STUNner is never going to be the bottleneck in the system. Obviously, if you have the needed computational power under your cluster.
+
+Scaling-up the number of instances in the `stunner` deployment should be done based on the CPU usage of the replicas. As it was mentioned in the [section above](#scaling-with-stunner) the `HorizontalPodAutoscaler` is using the requested resources to determine when to scale up.
```

Comment:
section above -> Section 'Scaling with STUNner'  : it is recommended to avoid relative refs in docs. In case of text reorg, relative refs are a horror.